packages/styles/dialog.css

```diff
@@ -110,10 +110,13 @@
   justify-content: center;
   border: none;
   background-color: white;
+  height: fit-content;
```

Comment:
I don't think has wide support yet. https://caniuse.com/?search=fit-content

What does this look like when `fit-content` is not supported?

Reply:
I changed it into "auto". So far the behavior is the same, and when heading is too long it will wrap. Let me know if there is other issues.